src/presets/vercel/utils.ts

```diff
@@ -13,15 +13,30 @@ import { isTest } from "std-env";
 
 // https://vercel.com/docs/build-output-api/configuration
 
+// https://vercel.com/docs/functions/runtimes/node-js/node-js-versions
+const SUPPORTED_NODE_VERSIONS = [16, 18, 20, 22];
```

Comment:
16 is not supported by nitro

```suggestion
const SUPPORTED_NODE_VERSIONS = [18, 20, 22];
```